doc/api/cli.md

```diff
@@ -1577,6 +1577,42 @@ on the number of online processors.
 If the value provided is larger than V8's maximum, then the largest value
 will be chosen.
 
+### `--watch`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Starts Node.js in watch mode.
+When in watch mode, changes in the watched files cause node to restart.
+By default, watch mode will watch the entry point
+and any required or imported module.
+Use `--watch-path` to specify what paths to watch.
```

Comment:
```suggestion
When in watch mode, changes in the watched files cause the Node.js process to
restart.

By default, watch mode will watch the entry point and any required or imported
module.

Use `--watch-path` to specify what paths to watch.
```

Reply:
This should clarify... does the current process exit and restart with a new pid? Does the existing process just reload? What happens to any worker threads or child processes?